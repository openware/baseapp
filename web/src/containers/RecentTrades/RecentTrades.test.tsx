import { render } from '@testing-library/react';
import React from 'react';
import { TestComponentWrapper } from 'src/lib/test';
import { RecentTrades } from '..';

const defaultProps: {
    recentTrades: any[];
    currentMarket: undefined;
    userLoggedIn: boolean;
    currentPrice: undefined;
} = {
    recentTrades: [],
    currentMarket: undefined,
    currentPrice: undefined,
    userLoggedIn: true,
};

const renderComponent = () =>
    render(
        <TestComponentWrapper>
            <RecentTrades {...{ ...defaultProps }} />
        </TestComponentWrapper>,
    );

describe('RecentTradesComponent', () => {
    it('should render', () => {
        expect(renderComponent().container).toMatchSnapshot();
    });
});
