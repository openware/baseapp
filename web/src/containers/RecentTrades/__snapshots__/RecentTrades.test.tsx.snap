// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentTradesComponent should render 1`] = `
<ContextProvider
  value={
    Object {
      "store": Object {
        "@@observable": [Function],
        "dispatch": [Function],
        "getState": [Function],
        "replaceReducer": [Function],
        "subscribe": [Function],
      },
      "subscription": Object {
        "addNestedSub": [Function],
        "getListeners": [Function],
        "handleChangeWrapper": [Function],
        "isSubscribed": [Function],
        "notifyNestedSubs": [Function],
        "onStateChange": [Function],
        "trySubscribe": [Function],
        "tryUnsubscribe": [Function],
      },
    }
  }
>
  <Connect(injectIntl(Connect(RecentTradesComponent)))
    recentTrades={Array []}
    userLoggedIn={true}
  />
</ContextProvider>
`;
